src/apsbits/core/instrument_init.py

```diff
@@ -130,8 +145,18 @@ def namespace_loader(yaml_device_file, main=True):
     t0 = time.time()
 
     current_devices = oregistry.device_names
-
-    instrument.load(yaml_device_file)
+    # Convert pathlib.Path to string - this fixes the guarneri bug
```

Comment:
What's the guarneri bug? Instrument.load should accept any of `config_file: Path | str | IO`.